packages/http-caching-proxy/src/http-caching-proxy.ts

```diff
@@ -187,27 +196,43 @@ export class HttpCachingProxy {
     clientResponse: ServerResponse,
   ) {
     debug('Forward request to %s %s', clientRequest.method, clientRequest.url);
-    const backendResponse = await makeRequest({
-      resolveWithFullResponse: true,
-      simple: false,
-
-      method: clientRequest.method,
-      uri: clientRequest.url!,
-      headers: clientRequest.headers,
-      body: clientRequest,
-      timeout: this._options.timeout || undefined,
-    });
+    let backendResponse: AxiosResponse;
+    try {
+      backendResponse = await this._axios({
+        method: clientRequest.method as Method,
+        url: clientRequest.url!,
+        headers: clientRequest.headers,
+        data: clientRequest,
+        timeout: this._options.timeout || undefined,
+      });
 
-    debug(
-      'Got response for %s %s -> %s',
-      clientRequest.method,
-      clientRequest.url,
-      backendResponse.statusCode,
-      backendResponse.headers,
-    );
+      debug(
+        'Got response for %s %s -> %s',
+        clientRequest.method,
+        clientRequest.url,
+        backendResponse.status,
+        backendResponse.headers,
+        backendResponse.data,
+      );
+    } catch (err) {
+      debug(
+        'Error sending request %s %s',
```

Comment:
Is this debug statement needed? I believe the error will be logged by `onerror` handler, no need to log it again here.